docs/api.rst

```diff
@@ -213,3 +213,9 @@ Exporting data
 
 .. autoenum:: conflowgen.ExportFileFormat
     :members:
+
+Data summary caching
```

Comment:
Please add this class to the section "Working with reports" inside the API file.

Reply:
See [Move Cache to 'Working with Reports' in docs](https://github.com/1kastner/conflowgen/pull/183/commits/cac53063548fbbefc0b0014932b95b154a543a49)